exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/transactions/TransactionJsonMessage.java

```diff
@@ -18,7 +18,7 @@
 
 /**
  * Data of a transaction message in JSON. Matches the
```

Comment:
I think "matches Exonum format" part is obsolete (they are now passed as hex).

Do we still need this class?

Reply:
The downsides of suppressing warnings :-)

Reply:
:arrow_up_small: — it does not seem to be used.